gui/symfony-docker/src/Controller/EleveurController.php

```diff
@@ -75,21 +77,18 @@ public function naissance(Request $request,
     if ($form->isSubmitted() && $form->isValid()) {
         try {
             // dd($form->getData());
-            $metadata = $this->hardwareService->metadataTemplate((int)$form->getData()->getWeight(),
+            $metadata = $this->blockChainService->metadataTemplate((int)$form->getData()->getWeight(),
                                                                 (int)$form->getData()->getPrice(),
                                                                 $form->getData()->getDescription(),
                                                                 $form->getData()->getGenre()
                                                             );
-            $response = $this->hardwareService->mintResource(1,1, ['metadata' => $metadata]);
+            $response = $this->blockChainService->mintResource(1,1, ['metadata' => $metadata]);
             $responseArray = json_decode($response, true);
-            // dd($responseArray);
-            // $this->addFlash('success', 'La naissance de votre animal a bien été enregistrée ! NFC : ' . $responseArray[1]['tokenId']);
         } catch (UniqueConstraintViolationException){
-            $this->addFlash('error', 'Le tag NFC est déjà utilisé');
+            $this->addFlash('error', 'Le tag NFT est déjà utilisé');
```

Comment:
Ce message d'erreur n'a rien à voir, à remplacer par :    $this->addFlash('error', 'Le NFT existe déjà');